ACPs/77-reinventing-subnets/README.md

```diff
@@ -109,21 +76,74 @@ type SubnetValidator struct {
     ChangeOwner fx.Owner `json:"changeOwner"`
 }
 
-type RecoverSubnetTx struct {
+type ConvertSubnetTx struct {
     // Metadata, inputs and outputs
     BaseTx
-    // Validator set for the Subnet
+    // ID of the Subnet to transform
+    // Restrictions:
+    // - Must not be the Primary Network ID
+    Subnet ids.ID `json:"subnetID"`
+    // Chain where the Subnet manager lives
+    ChainID ids.ID `json:"chainID"`
```

Comment:
```suggestion
    // BlockchainID where the L1 manager lives
    ChainID ids.ID `json:"chainID"`
```
ChainID (an integer set in genesis that has no functional use afaiaa) is often confused with _this instance_ of ChainID (the BlockchainID of the L1 derived from the txn hash of `CreateChainTx`).

Reply:
👍 FWIW, the EVM `chainID` set in subnet-evm genesis files is used by those chains to enforce replay protection across any other EVM chains. This is why transactions sent from accounts on Ethereum can't be broadcasted on other EVM chains.